app/src/test/java/br/com/concrete/tentacle/repositories/SharedPrefRepositoryTest.kt

```diff
@@ -0,0 +1,71 @@
+package br.com.concrete.tentacle.repositories
+
+import br.com.concrete.tentacle.features.MainActivity
+import br.com.concrete.tentacle.mock.sessionForPreference
+import br.com.concrete.tentacle.mock.sessionKey
+import br.com.concrete.tentacle.mock.string
+import br.com.concrete.tentacle.mock.stringExpectedWhenThereIsNoOne
+import br.com.concrete.tentacle.mock.stringKey
+import junit.framework.Assert.assertEquals
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.koin.standalone.StandAloneContext
+import org.koin.standalone.inject
+import org.koin.test.KoinTest
+import org.robolectric.Robolectric
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
```

Comment:
Acho que não seria necessário utilizar o Roboletric, voces poderiam mockar o SharedPref ou ver algo com relação a nova versão do koin que voces estão utilizando. 

https://stackoverflow.com/questions/35105545/how-to-mock-a-sharedpreferences-using-mockito

https://proandroiddev.com/testing-with-koin-ade8a46eb4d